epaint/src/text/text_layout_types.rs

```diff
@@ -257,6 +257,54 @@ impl TextFormat {
 
 // ----------------------------------------------------------------------------
 
+#[derive(Clone, Debug, PartialEq)]
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+pub struct TextWrapping {
+    /// Try to break text so that no row is wider than this.
+    /// Set to [`f32::INFINITY`] to turn off wrapping.
+    /// Note that `\n` always produces a new line.
+    pub max_width: f32,
+
+    /// Maximum amount of rows the text should have.
+    /// Set to `0` to disable this.
+    pub max_lines: usize,
```

Comment:
```suggestion
    pub max_rows: usize,
```

so as not to confuse it with a paragraph (that which is ended by the newline character `'\n'`)